src/header/Header.tsx

```diff
@@ -31,7 +30,7 @@ const Header = ({
   containerProps = {},
 }: HeaderProps) => {
   const intl = useIntl();
-  const libraryHref = useHref('/library/:libraryId');
+  const waffleFlags = useSelector(getWaffleFlags);
```

Comment:
Note: in backporting this fix, I also included the waffle flag checks added by https://github.com/openedx/frontend-app-authoring/commit/f9ef00e29f27881c44ee6ee4f36c667e26d58e56#diff-1c1e7c5912d6ff798a72899cde884c54211549e22fec1f469c8a5a4a854819e1

This was not strictly a bugfix (as the sumac backend still handles these redirects), but it seemed innocuous enough to include here to resolve the cherry-picked conflict. Happy to revert this part of the change if anyone is concerned.